cognite/neat/_graph/transformers/_classic_cdf.py

```diff
@@ -295,7 +302,22 @@ def __init__(
     ?entity classic:externalId "{external_id}" .
 }}"""
 
+    @staticmethod
+    def create_lookup_entity(graph: Graph, namespace: Namespace) -> Callable[[str, str], URIRef]:
+        @lru_cache(maxsize=10_000)
```

Comment:
I am not sure why would this help as the sparql query is executed anyhow yielding you all results back

Reply:
what do you mean when you say `lookup_entity` ? you are getting rdf:type Relationship but only those that have `classix:externalId` as property?

Reply:
Name `create_lookup_entity` is not concrete, what you are actually doing here is to get ID of instance by supplying value for its externalid property. Again my mistake not throughly reviewing your previous PR.

Reply:
This is new, so you have not seen it before. Renamed it now.